lib/lotus/view/configuration.rb

```diff
@@ -388,6 +409,7 @@ def reset!
         root             DEFAULT_ROOT
         default_encoding DEFAULT_ENCODING
 
+        @partials   = Hash.new
```

Comment:
I've added unit tests for `Configuration`.

Also changed the `@partials` hash as suggested, though it had some knock-on effects in the lookup logic which relied on `@partials[missing_key]` returning a nil. So I'm not sure I really like this change because lookups can mutate the cache.